jersey-servers/build.gradle

```diff
@@ -15,4 +15,5 @@ dependencies {
     testCompile "io.dropwizard:dropwizard-testing"
     testCompile "junit:junit"
     testCompile "org.hamcrest:hamcrest-all"
+    testCompile "org.mockito:mockito-core"
```

Comment:
You have been relying on transitive mockito-core import from dropwizard that's no longer there

Reply:
cheers